build.gradle

```diff
@@ -124,19 +124,18 @@ subprojects { subproject ->
 	// dependencies that are common across all java projects
 	dependencies {
 		compileOnly "com.google.code.findbugs:jsr305:$googleJsr305Version"
-		testCompile ("junit:junit:$junit4Version") {
-			exclude group: 'org.hamcrest', module: 'hamcrest-core'
-		}
 		testCompile "org.apache.logging.log4j:log4j-core:$log4jVersion"
 		testCompile "org.hamcrest:hamcrest-all:$hamcrestVersion"
 		testCompile ("org.mockito:mockito-core:$mockitoVersion") {
 			exclude group: 'org.hamcrest', module: 'hamcrest-core'
 		}
+		testCompile "org.mockito:mockito-junit-jupiter:$mockitoVersion"
```

Comment:
What is cool in this? Why do we need this extra dep?
Thanks

Reply:
`@ExtendWith(MockitoExtension.class)` (for classes with `@Mock`, `@Spy`.